content/en/guides/hosting/data-security/secure-storage-connector.md

```diff
@@ -7,118 +7,236 @@ title: Bring your own bucket (BYOB)
+            "arn:aws:s3:::<cw_bucket>/*"
+          ]
+        }
+      ]
+    }
+    ```
 
-From W&B instance in GCP or AWS to a bucket in Azure:
-```text
-az://:<urlEncodedAccessKey>@<storageAccountName>/<containerName>
-```
+    <!-- </details>
 
-From W&B instance in AWS or Azure to a bucket in GCP:
-```text
-gs://<serviceAccountEmail>:<urlEncodedPrivateKey>@<bucketName>
-```
+    <details>
+      <a id="coreweave-multi-tenant-cloud-bucket-policy"></a><summary><b>Multi-tenant Cloud</b> bucket policy</summary>
 
-{{% alert %}}
-Because it is hosted in GCP, [SaaS Cloud]({{< relref "/guides/hosting/hosting-options/saas_cloud.md" >}}) does not support connectivity to S3-compatible storage for team-level storage, including cross-cloud connectivity. SaaS Cloud does not have access to the access keys and environment variables used to configure [Dedicated cloud]({{< relref "/guides/hosting/hosting-options/dedicated_cloud.md" >}}) and [Self-managed]({{< relref "/guides/hosting/hosting-options/self-managed.md" >}}) instances.
-{{% /alert %}}
-</details>
+    **Multi-tenant Cloud only**:
+    For team level storage in Multi-tenant Cloud, obtain your W&B organization ID, which is required in your bucket policy for team level storage:
+      1. Log in to W&B as a user with the `admin` role.
+      1. Click the icon at the top left to open the left navigation, then click **Create a team to collaborate**.
+      1. Click **External storage**, then set **Cloud Provider** to **CoreWeave**.
+      1. Near the bottom of the dialog, make a note of the **W&B organization ID**.
+      1. Keep this page open. Later, you will use it to create the new team.
+    
+    Replace:
+    - In `Resource`, each instance of `<cw_bucket>` with the CoreWeave bucket name.
+    - In `Principal` and `Condition`, each instance of `<wb-org-id>` with your W&B organization ID.
 
+    ```json
+    {
+      "Version": "2012-10-17",
+      "Id": "WandBAccess",
+      "Statement": [
+        {
+          "Sid": "WAndBAccountAccess",
+          "Effect": "Allow",
+          "Principal": { "CW": "arn:aws:iam::<wb-org-id>:*" },
+            "Action" : [
+              "s3:GetObject*",
+              "s3:GetEncryptionConfiguration",
+              "s3:ListBucket",
+              "s3:ListBucketMultipartUploads",
+              "s3:ListBucketVersions",
+              "s3:AbortMultipartUpload",
+              "s3:DeleteObject",
+              "s3:PutObject",
+              "s3:GetBucketCORS",
+              "s3:GetBucketLocation",
+              "s3:GetBucketVersioning"
+            ],
+            "Resource": [
+              "arn:aws:s3:::<cw_bucket>",
+              "arn:aws:s3:::<cw_bucket>/*"
+            ],
+            "Condition": {
+			        "StringLike": {
+				        "wandb:OrgID": [
+					       "<wb-org-id>"
+				        ]
+			        }
+		        }
+        }
+      ]
+    }
+    ```
 
-## Cloud storage in same cloud as W&B platform
+    </details>-->
```

Comment:
There is a trailing "-->" showing up in the docs: 

<img width="799" alt="Screenshot 2025-06-27 at 1 19 42 PM" src="https://github.com/user-attachments/assets/b675d677-989a-4c04-9bde-84a5590d9a48" />

Reply:
Yeah you are right, hopefully fixed adfter this push!